lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
```

Comment:
IIRC, there was another proposal that included a `view` parameter as a reference to the database concept (a named stored query). I think @andy-armstrong was the one who pointed me in that direction. If you can come up with something decidedly better, I don't think it would be out of scope to change the proposal. `view` is already different than the original proposal, and you won't get any pushback for parameter names as long as they are well justified.